src/dorsal.js

```diff
@@ -226,12 +226,16 @@ DorsalCore.prototype._wireElements = function(nodes, plugins, deferred) {
  */
 DorsalCore.prototype._wireElement = function(el, plugins, deferred) {
     var self = this,
+        dfd = new DorsalDeferred(),
         log = new DorsalLog(this.DEBUG);
 
+    deferred.when(dfd);
```

Comment:
I don't think we should have this here. Let's instead collect all the dfd's returned from _wireElement and pass them into deferred.when.

We shouldn't be passing deferred into _wireElement either. Basically, let's not let _wireElement know about some global state.